ts/features/itwallet/machine/credential/guards.ts

```diff
@@ -2,24 +2,36 @@ import { pipe } from "fp-ts/lib/function";
 import * as O from "fp-ts/lib/Option";
 import { ItwSessionExpiredError } from "../../api/client";
 import { isWalletInstanceAttestationValid } from "../../common/utils/itwAttestationUtils";
+import { useIOStore } from "../../../../store/hooks";
+import { itwLifecycleIsValidSelector } from "../../lifecycle/store/selectors";
 import { Context } from "./context";
 import { CredentialIssuanceEvents } from "./events";
 import { CredentialIssuanceFailureType } from "./failure";
 
-export const createCredentialIssuanceGuardsImplementation = () => ({
-  isSessionExpired: ({ event }: { event: CredentialIssuanceEvents }) =>
-    "error" in event && event.error instanceof ItwSessionExpiredError,
+export const createCredentialIssuanceGuardsImplementation = () => {
+  // eslint-disable-next-line react-hooks/rules-of-hooks
+  const store = useIOStore();
```

Comment:
You can't use hooks outside of components or other hooks, you need to pass the `store` as an argument.
```suggestion
export const createCredentialIssuanceGuardsImplementation = (store: ReturnType<typeof useIOStore>) => {
```

Reply:
Yes, you're right! Addressed in [0828a30](https://github.com/pagopa/io-app/pull/6580/commits/0828a300826f23e9906e466a5ed4ec313f523397)